### PR TITLE
[4.4] Fix missing $db->quoteName in FieldLayoutField

### PR DESCRIPTION
**Pull request by LadySolveig**
Pull Request for Comment https://github.com/joomla/joomla-cms/pull/43829#pullrequestreview-2201148476 .

### Summary of Changes

Add missing $db->quoteName in FieldLayoutField

### Testing Instructions

Create override `com_fields` -> `fields`

grafik

Rename the layout file e. g. `test.php`

grafik

`Content` -> `Fields`-> `New` -> `Tab: Options` -> and scroll to bottom

you should see your new layout override in the Layout Dropdown

### Actual result BEFORE applying this Pull Request

grafik

### Expected result AFTER applying this Pull Request

Nothing should be changed.

grafik

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [X] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [X] No documentation changes for manual.joomla.org needed

### CODE REVIEW

File: administrator/components/com_fields/src/Field/FieldLayoutField.php [hunk 1]
```diff
@@ -56,9 +56,9 @@ protected function getInput()
             // Build the query.
             $query->select('element, name')
                 ->from('#__extensions')
-                ->where('client_id = 0')
-                ->where('type = ' . $db->quote('template'))
-                ->where('enabled = 1');
+                ->where($db->quoteName('client_id') . '=' . $db->quote('0'))
+                ->where($db->quoteName('type') . '=' . $db->quote('template'))
+                ->where($db->quoteName('enabled') . '=' . $db->quote('1'));
```
Comment (Quy) [inline administrator/components/com_fields/src/Field/FieldLayoutField.php]:
```suggestion
                ->where($db->quoteName('client_id') . ' = 0')
                ->where($db->quoteName('type') . ' = ' . $db->quote('template'))
                ->where($db->quoteName('enabled') . ' = 1');
```

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on e9a6d1b1191081c86d695df09f6a8c94a9fd7c96<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43851">issues.joomla.org/tracker/joomla-cms/43851</a>.</sub>

File: administrator/components/com_fields/src/Field/FieldLayoutField.php (hunk 1 above)
Comment (LadySolveig) [reply administrator/components/com_fields/src/Field/FieldLayoutField.php]: Thanks!

**Comment by chmst** [comment]
I have tested this item :white_check_mark: successfully on e9a6d1b1191081c86d695df09f6a8c94a9fd7c96<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43851">issues.joomla.org/tracker/joomla-cms/43851</a>.</sub>

**Comment by chmst** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43851">issues.joomla.org/tracker/joomla-cms/43851</a>.</sub>

**Review by richard67 — APPROVED** [review]

**Pull request merged**